src/strategies/passport/passport-apple-oauth2/apple-auth-verify.provider.ts

```diff
@@ -9,8 +9,6 @@ import {VerifyFunction} from '../../types';
 export class AppleAuthVerifyProvider
   implements Provider<VerifyFunction.AppleAuthFn>
 {
-  constructor() {}
```

Comment:
why are constructors removed 
how will class be initialised without constructors

Reply:
It was there in sonar isssue, to remove empty constructors.